src/reducers/activeUser.js

```diff
@@ -0,0 +1,10 @@
+export default function(state=null, action) {
```

Comment:
In this case might be okay, but have a look how to write reducers according to documentation
https://redux.js.org/basics/reducers